crypto/fipsmodule/CMakeLists.txt

```diff
@@ -163,10 +163,21 @@ if (S2N_BIGNUM_DIR AND UNIX AND NOT APPLE)
     bignum_tomont_p384.S
     bignum_deamont_p384.S
     bignum_montmul_p384.S
+    bignum_montmul_p384_alt.S
```

Comment:
Is "alt" and abbreviation for "alternative". Maybe not the best name if there is a third "alternative" solution.

Reply:
Good thing we can influence s2n-bignum quite easily then ;)